### PR TITLE
chore(ui): upgrade to Clockface 2.0.0

### PR DESCRIPTION
**Pull request by alexpaxton**
- Upgrade to latest Clockface version
- Remove font files
- Import `variables.scss` from Clockface into local `modules.scss` file
- Remove redundant stylesheets (reset, fonts, typography, etc)
- Remove `PageTitleWithOrg`
- Update design of page headers throughout UI to conform to new header / control bar pattern

- [x] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
- [x] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [x] Rebased/mergeable
- [ ] Tests pass
- [ ] http/swagger.yml updated (if modified Go structs or API)
- [ ] Documentation updated or issue created (provide link to issue/pr)
- [ ] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: ui/src/shared/components/AddResourceDropdown.tsx [hunk 1]
```diff
@@ -37,7 +37,7 @@ export default class AddResourceDropdown extends PureComponent<Props> {
     const {titleText, status} = this.props
     return (
       <Dropdown
-        style={{width: `${160}px`}}
+        style={{width: `${190}px`}}
```
Comment (121watts) [inline ui/src/shared/components/AddResourceDropdown.tsx]: this can just be `'190px'`. no need to use string interpolation.

Comment (alexpaxton) [reply ui/src/shared/components/AddResourceDropdown.tsx]: Good catch

**Comment by russorat** [comment]
testing this branch out, very cool.

Noticed the drop downs for variable values on the dashboards don't update.
image

**Comment by russorat** [comment]
Also, looks like the axis text on the graphs isn't formatted correctly.
image

**Comment by russorat** [comment]
drop down issue is actually in master: https://github.com/influxdata/influxdb/issues/17123

**Pull request merged**